extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -698,8 +698,8 @@ export class GmailElementReplacer implements WebmailElementReplacer {
               }
               if (typeof cache === 'undefined') {
                 try {
-                  const [contact] = await ContactStore.get(undefined, [email]);
-                  if (contact && contact.pubkey) {
+                  const contactWithPubKeys = await ContactStore.getOneWithAllPubkeys(undefined, email);
```

Comment:
need to think this through. Should we try to lookup a pubkey if we only have a revoked version in the cache?...